src/ert/scheduler/job.py

```diff
@@ -117,8 +116,7 @@ async def _submit_and_run_once(self, sem: asyncio.BoundedSemaphore) -> None:
         except asyncio.CancelledError:
             await self._send(State.ABORTING)
             await self.driver.kill(self.iens)
-            with suppress(asyncio.CancelledError):
-                await self.returncode
+            self.returncode.cancel()
```

Comment:
The reason we did `await self.returncode` is to ensure that the job is dead before we say `State.ABORTED`. Is it reasonable for this to be fire-and-forget?

Reply:
Yes, there is. If an exception in Polling then `await self.returncode` will hang (while we cal `job_task.cancel()`) which is treated here.